frontend/src/resources/LoginForm.css

```diff
@@ -0,0 +1,21 @@
+.input-field {
+    margin-bottom: 2%;
+}
+
+.input-container {
+    align-items: center;
+    width: 40%;
+    margin-left: 30%;
+}
+
+.form-container {
+    align-content: center;
+}
+
+.create-account-container {
+    margin: 1em;
```

Comment:
Do we know exactly what 1em is. I don't but if we have control over this then it's okay. I normally like working in pixels and and percentages where necessary but yeah that's personal preference.

Reply:
1em has been changed to 30px.